src/_premake_init.lua

```diff
@@ -757,6 +757,7 @@
 			"C90",
 			"C99",
 			"C11",
+			"C17",
```

Comment:
This should probably be done in a separate PR - I think it's far more likely `gmake2` users are going to want to use this than VS users. Similarly, there should be the `gnu17` option too.

Reply:
I was just getting all the language flags for MSVC up to date in their mappings. I'd be happy to do gmake2 in a separate PR.

Reply:
Yeah that's fair enough, I'll resolve this comment.